src/app/dashboard/revalidation-form/revalidation-form.component.scss

```diff
@@ -0,0 +1,9 @@
+.revalidation-form {
+  min-width: 150px;
+  max-width: 500px;
+  width: 100%;
+}
+
+.full-width {
```

Comment:
there is already a css class in utilities `.w-100 ` that does the same thing

Reply:
done